modules/tinymce/src/plugins/codesample/main/ts/ui/Dialog.ts

```diff
@@ -28,7 +29,8 @@ const open = (editor: Editor): void => {
         {
           type: 'textarea',
           name: 'code',
-          label: 'Code view'
+          label: 'Code view',
+          spellcheck: Options.shouldBrowserSpellcheck(editor),
```

Comment:
I think we should hardcode this to `false` since spellchecking source code is never going to work well and the `browser_spellcheck` option also toggles spellchecking inside the rich text editor and that might be problematic to enable while also having say our premium spellchecker enabled.
I think also we should disable this for the source code editor dialog since that is also mixed source code and plain text. If we get pushback on this we can re-introduce a `sourcecode_spellcheck` option or something that both these dialogs can use.